src/components/utils.js

```diff
@@ -0,0 +1,4 @@
+export function isContentManager() {
+  const roles = ["CM", "SA"];
+  return roles.some((value) => value === localStorage.getItem("role"));
+}
```

Comment:
`utils` shouldn't reside inside `components` create a separate directory for it.

`roles` seems to be a constant value move it outside this function. Reading from `localSrorage` is an expensive operation you can read the value from `localStorage` and save it in a variable to compare with other values.
```suggestion
const CONTENT_MANAGER_ROLES = ["CM", "SA"];

export function isContentManager() {
  const current_user_role = localStorage.getItem("role");
  return CONTENT_MANAGER_ROLES.some((role) => role === current_user_role);
}
```